package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "formifly",
-  "version": "2.5.0",
+  "version": "2.6.0",
```

Comment:
Not sure if this should even be a major version bump. I think according to semver dependency updates are "just" minor version bumps if our own API doesn't change?